tests/components/arlo/test_sensor.py

```diff
@@ -20,7 +22,14 @@ def _get_named_tuple(input_dict):
 def _get_sensor(name="Last", sensor_type="last_capture", data=None):
     if data is None:
         data = {}
-    return arlo.ArloSensor(name, data, sensor_type)
+    for entry in SENSOR_TYPES:
```

Comment:
```
sensor_entry = next(sensor_entry for sensor_entry in SENSOR_TYPES if sensor_entry.key == sensor_type)
return arlo.ArloSensor(data, sensor_entry)
```
Maybe?

Reply:
How elegant, I have noted that for future use! 

Done.